### PR TITLE
added gitUrl to analytics scan

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

**Comment by rupalibehera** [comment]
You cannot test this until you have the latest analytics plugin https://github.com/fabric8io/openshift-jenkins-s2i-config/pull/177 merged right ?

Let's hold on merging unless we have the new plugin merged is that ok ?

File: vars/bayesianScanner.groovy [hunk 1]
```diff
@@ -16,7 +16,8 @@ def call(body) {
             try {
                 sh 'mvn io.github.stackinfo:stackinfo-maven-plugin:0.2:prepare'
                 retry(3) {
-                    def response = bayesianAnalysis url: 'https://bayesian-link'
+                    def project = flow.getGitHubProject()
+                    def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: "git@github.com:${project}.git"
```
Comment (sthaha) [inline vars/bayesianScanner.groovy]: why is this using git@github.com instead of https://github.com/.. / ?

Comment (sthaha) [reply vars/bayesianScanner.groovy]: Have you see if flow already provides link to GH url?

**Comment by yzainee** [comment]
@sthaha yes it provides the github url..
from my logs "Git URL is git@github.com:yzainee/hellovertx11.git" . As for your query on https vs git@github, I had asked Sam for the format and he was fine with this format. Hence i kept it like this.

**Comment by sthaha** [comment]
@yzainee https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-urls-recommended in case you didn't know

**Pull request merged**